### ISSUE TITLE
When raising things to powers "error: division is not defined in this ring"

### ISSUE THREAD

**Issue by kschwede** [Macaulay2/M2#629]
Hi,

Pedro Teixeira and I recently noticed the following bug.

R = ZZ/11[x]
x^(11^13)

stdio:12:2:(3): error: division is not defined in this ring

some powers cause this error, some do not.  For example
x^(11^14)
simply produces
o15 = x^1710172809

This appears to impact versions of Macaulay2 from at least 1.8.2 to a version I compiled last week.

**Comment by DanGrayson** [comment]
Mike figured out why this is happening: 11^13 fits into a 64 bit integer, but when accidentally truncated to 32 bits, it gives a negative number:
```

i9 : 11^13 // 2^64

o9 = 0

i10 : 11^13 % 2^32

o10 = 4059985979

i11 : 2^31

o11 = 2147483648

i12 : 11^14 // 2^64

o12 = 0

i13 : 11^14 % 2^32

o13 = 1710172809

i14 : 2^31

o14 = 2147483648
```

The fix is coming soon.

**Comment by mikestillman** [comment]
Fixed in commit d36cb3ca2d5c6
